engine/crates/composition/src/composition_ir.rs

```diff
@@ -32,8 +32,12 @@ pub(crate) struct CompositionIr {
 
     pub(crate) strings: StringsIr,
     pub(crate) fields: Vec<FieldIr>,
-    pub(crate) union_members: BTreeSet<(subgraphs::StringId, subgraphs::StringId)>,
+    pub(crate) union_members: BTreeSet<(federated::StringId, federated::StringId)>,
     pub(crate) resolvable_keys: Vec<KeyIr>,
+
+    /// Fields on implementers of an interface entity's implementers that are contributed by other
+    /// subgraphs.
```

Comment:
Reformulate. "Fields of an interface entity that are contributed by other subgraphs and must be added to the interface's implementers in the federated schema".